_artifacts/document-attributes-dm.adoc

```diff
@@ -42,4 +42,6 @@
 :CENTRAL_CAPITAL_UNDER: DECISION_CENTRAL
 :CENTRAL_ONEWORD: decisioncentral
 
+:KAFKA: Red Hat AMQ Streams
```

Comment:
@mramendi , we need to clarify some terminology here.  I already went through this for Kogito, and now that I see it here, I chatted with Ben Hardesty from AMQ to confirm. From my understanding on Kogito and now from Ben, you have it right for the platforms, but not for components of the platforms, such as messages, clusters, etc., which are still known as "Kafka messages", "Kafka clusters", even in enterprise. The exception is "AMQ Streams server" which is accurate I believe.

So to summarize, it's like this:

- AMQ Streams on RHEL roughly = Apache Kafka in community, really just based on it like you've said in the docs. So these attributes you have here are correct to use **for the platforms themselves**, but not for all components.
- AMQ streams on OpenShift = Strimzi in community, which is also based on Kafka but has that distinct community name. This doesn't seem to apply in this doc, but if it does, then the attributes will need to be revisited. I do see a link to both types of AMQ at the end of the doc as additional resources.
- Kafka messages, Kafka clusters, etc., are the correct terms for both enterprise and community. We shouldn't say AMQ Streams messages, AMQ Streams clusters, etc.
- AMQ Streams server is correct however, from what I understand.

So the only need change in this doc that I see is that all instances of `{KAFKA} messages` need to be changed to just `Kafka messages`, no attribute, to ensure it appears that way in both outputs. This includes all references used in the headings too. Only references to the platform and server should use the attribute, i.e., `{KAFKA} ifdef... based on Apache Kafka endif  is a streaming platform` is correct, but `{KAFKA} messages` isn't.